quinn/src/runtime.rs

```diff
@@ -25,6 +25,12 @@ pub trait Runtime: Send + Sync + Debug + 'static {
     fn now(&self) -> Instant {
         Instant::now()
     }
+    /// Look up the current [`SystemTime`]
```

Comment:
style: empty line between definitions

Reply:
Then I shall fix it for the others in this file too.